File: js/sumPairs.js
function sumPairs(arr, target) {
  let res = [];
  let sum = 0;
  for(let i = 0; i < arr.length; i++) {
    for(let k = i + 1; k < arr.length; k++) {
      if(arr[i] + arr[k] === target) {
       res.push([arr[i], arr[k]]);
      }
    }
  }
  if(res.length === 0) {
    return('unable to find pairs');
  }
  return res;
};

console.log(sumPairs([1,2,3,4,5], 9));
console.log(sumPairs([1,2,3,4,5], 7));
console.log(sumPairs([3, 1, 5, 8, 2], 27));

File: python/sum_pairs.py
def sum_pairs(arr , target):
    no_pairs='unable to find pairs'
    res= []
    sum=0
    for index1,i in enumerate(arr):
            for index2,k in enumerate(arr):
                if arr[index1] >=arr[index2]:
                        continue
                if arr[index1]+arr[index2]== target:
                    res.append( [arr[index1],arr[index2]])
                   
    if len(res) == 0:
        return no_pairs
    return res

